Packs/AWS-IAMIdentityCenter/pack_metadata.json

```diff
@@ -1,8 +1,8 @@
 {
     "name": "AWS - IAM Identity Center",
     "description": "AWS IAM Identity Center\n\nAuthor: Sameh El-Hakim\n\nWith AWS IAM Identity Center (successor to AWS Single Sign-On), you can manage sign-in security for your workforce identities, also known as workforce users. IAM Identity Center provides one place where you can create or connect workforce users and manage their access centrally across all their AWS accounts and applications. IAM Identity Center is the recommended approach for workforce authentication and authorization in AWS, for organizations of any size and type.",
-    "support": "community",
-    "currentVersion": "1.0.1",
+    "support": "xsoar",
+    "currentVersion": "1.0.2",
     "author": "Sameh El-Hakim",
```

Comment:
please change the support to xsoar and check what are all the required changes you need to make to the metadata file to adopat as xsoar support.